packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/edit-connector/edit-connector.html

```diff
@@ -0,0 +1,91 @@
+<oui-back-button
+    data-previous-page="{{:: 'pci_databases_connectors_edit_back' | translate }}"
+    data-on-click="$ctrl.goBack()"
+></oui-back-button>
+
+<!-- error messages -->
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+        <h2 ng-bind="$ctrl.availableConnector.name" class="mb-0"></h2>
+        <span class="oui-paragraph">
+            Version: {{$ctrl.availableConnector.version}} | Author:
+            {{$ctrl.availableConnector.author}} |
+            <a
+                ng-href="{{$ctrl.availableConnector.documentationUrl}}"
+                target="_blank"
+                rel="noopener"
+                >docs</a
+            >
+        </span>
+    </div>
+    <oui-button variant="primary" size="l" ng-click="$ctrl.updateConnector()">
```

Comment:
For consistency:
```suggestion
    <oui-button variant="primary" size="l" on-click="$ctrl.updateConnector()">
```

Reply:
Changed here: https://github.com/ovh/manager/pull/6579/commits/deefd28e570922645a1df5eb36318b0d30427fb2